config/config.mainnet.toml

```diff
@@ -75,4 +75,6 @@ enables = [
     "Transaction",
     "Validator",
     "ValidatorStats",
+    "NFT",
+#    "CryptoComNFT",
```

Comment:
CryptoComNFT will be enabled when?

Reply:
"CryptoComNFT" is dedicated for Crypto.com based NFT. So by default it is disabled. If interested party are running Crypto.com compatible NFT formats they can disable "NFT" and enable it instead.

Reply:
The logic's here
https://github.com/crypto-com/chain-indexing/pull/413/files#diff-36233cad5c04d2ed21a0945e0bb73cadb627342617e62ed07ca27184b8bf1d84R48